src/gui/connectionvalidator.cpp

```diff
@@ -312,13 +315,12 @@ void ConnectionValidator::fetchUser()
     job->start();
 }
 
-bool ConnectionValidator::setAndCheckServerVersion(const QString &version)
+bool ConnectionValidator::setAndCheckServerVersion(const QString &version, const QString &serverProduct)
```

Comment:
Naming wise you may want to call this method `setAndCheckServerInfo` now.

Reply:
![image](https://user-images.githubusercontent.com/200626/168764865-2d86fbc6-7ea1-4fa4-a290-26faf8da2e19.png)